### PR TITLE
Since now on main branch provides Jakarta EE API version only

### PR DESCRIPTION
**Pull request by ropalka**
https://issues.redhat.com/browse/EJBCLIENT-441
https://issues.redhat.com/browse/EJBCLIENT-443
https://issues.redhat.com/browse/EJBCLIENT-444

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -59,399 +53,319 @@
...
-                    </executions>
-                </plugin>
-            </plugins>
-        </pluginManagement>
+        <resources>
+            <resource>
+                <directory>src/main/java</directory>
+                <includes>
+                    <include>**/Version.properties</include>
+                </includes>
```
Comment (chengfang) [inline pom.xml:84]: the above element does not match any files. The version.properties is covered by the next element. Any reason it's added here?

File: pom.xml [hunk 2]
```diff
@@ -59,399 +53,319 @@
...
+            <artifactId>jakarta.resource-api</artifactId>
+            <version>${version.jakarta.resource.jakarta-resource-api}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.jboss</groupId>
+            <artifactId>jboss-transaction-spi</artifactId>
+            <version>${version.org.jboss.jboss-transaction-spi}</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (chengfang) [inline pom.xml:336]: we shouldn't need the above dep; its jakarta variant dep is already included below.

**Comment by chengfang** [comment]
@ropalka  can you please rebase? I fixed CI setup and should give better results.

File: pom.xml (hunk 1 above)
Comment (ropalka) [reply pom.xml]: No specific reason. I used pom.xml from former JBoss EJB Client releases as a template. Fixed @chengfang

File: pom.xml (hunk 2 above)
Comment (ropalka) [reply pom.xml:329]: You are right. Fixed @chengfang

**Review by chengfang — APPROVED** [review]

**Pull request merged**